airbyte-db/lib/src/main/java/io/airbyte/db/Databases.java

```diff
@@ -131,9 +131,9 @@ public static Database createDatabase(final String username,
                                         final String jdbcConnectionString,
                                         final String driverClassName,
                                         final SQLDialect dialect,
-                                        final String connectionProperties) {
+                                        final Map<String, String> connectionProperties) {
```

Comment:
Are there some needed updates on this code https://github.com/airbytehq/airbyte/blob/c7b8db702e8c0505f624101905ef778a12ae10f0/airbyte-integrations/connectors/source-oracle-strict-encrypt/src/test-integration/java/io/airbyte/integrations/source/oracle_strict_encrypt/OracleStrictEncryptSourceAcceptanceTest.java#L63 due to these changes?